watcher/watch.go

```diff
@@ -25,10 +26,16 @@ func init() {
 	}
 }
 
-func WatchRecursively(path string) error {
+func WatchRecursively(path string, ignoreDirs []string) error {
 	// Watch root directory and all subdirectories in the given path
 	err := filepath.Walk(path, func(filePath string, info os.FileInfo, err error) error {
 		if info.IsDir() {
+			for _, d := range ignoreDirs {
+				if d == info.Name() {
+					return filepath.SkipDir
```

Comment:
`info.Name()` returns the base name of the directory.

`filepath.Walk` will begin with the directory given as the `path` parameter.

I think this would cause the program to not watch _any_ directories if a directory in `.klausignore` has the same name as the top level directory (`path` argument).